doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-run-test-proc.adoc

```diff
@@ -8,7 +8,7 @@ After creating a test scenario template and defining the test scenarios, you can
 * To execute all the available test scenarios in your project inside multiple assets, in the upper-right corner of your project page, click *Test*.
 +
 .Run all the test scenarios from the project view
-image:AuthoringAssets/test-scenarios-run-all-tests-button.png[Run all the test scenarios from the project view]
+image::AuthoringAssets/test-scenarios-run-all-tests-button.png[Run all the test scenarios from the project view]
 +
 * To execute all the available test scenarios defined in a single `.scesim` file, at the top of the Test Scenario designer, click the *Run Test* image:AuthoringAssets/test-scenarios-run-test-icon.png[Run Test icon] icon.
```

Comment:
Minor suggestion - remove article and simplify:

s/To execute all the available test scenarios defined in a single `.scesim` file/To execute all available test scenarios defined in a `.scesim` file

Reply:
Done